server/TracyWorker.cpp

```diff
@@ -400,8 +400,8 @@ Worker::Worker( const std::string& program, const std::vector<ImportEventTimelin
         plot->type = PlotType::User;
         plot->format = v.format;
 
-        double min = v.data.begin()->first;
-        double max = v.data.begin()->first;
+        double min = v.data.begin()->second;
```

Comment:
This is supposed to be min and max value, not min and max timestamp, right?

Reply:
Yup.